metpy/plots/declarative.py

```diff
@@ -866,4 +870,36 @@ def _build(self):
         x, y, imdata = self.plotdata
         self.handle = self.parent.ax.contour(x, y, imdata, self.contours,
                                              colors=self.linecolor, linewidths=self.linewidth,
+                                             linestyles=self.linetype,
                                              transform=imdata.metpy.cartopy_crs)
+        if self.clabels:
+            cl = plt.clabel(self.handle, inline=1, fmt='%.0f', inline_spacing=2,
+                            use_clabeltext=True)
+
+
+@exporter.export
+class ContourFillPlot(Plot2D):
```

Comment:
Thoughts on name `FilledContourPlot`? I'm not sure.

Reply:
I like ```FilledContourPlot()``` better as it is more distinct from just ```ContourPlot()```